x/collector/keeper/collector.go

```diff
@@ -556,9 +591,13 @@ func (k *Keeper) WasmSetCollectorLookupTableQuery(ctx sdk.Context, appID, collec
 }
 
 func (k *Keeper) WasmSetAuctionMappingForApp(ctx sdk.Context, auctionMappingBinding *bindings.MsgSetAuctionMappingForApp) error {
+	result1, found := k.GetAuctionMappingForApp(ctx, auctionMappingBinding.AppMappingID)
 	var collectorAuctionLookup types.CollectorAuctionLookupTable
 	collectorAuctionLookup.AppId = auctionMappingBinding.AppMappingID
 	var assetIDToAuctionLookups []types.AssetIdToAuctionLookupTable
+	if found {
+		assetIDToAuctionLookups = result1.AssetIdToAuctionLookup
```

Comment:
Is This check correct ? can you re-verify it ?

Reply:
this is also reasoned above but the difference is this one is for wasm . both perform same functionality

Reply:
yes correct